sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -54,11 +58,6 @@
   from .slow_stream import get_varint_size
 # pylint: enable=wrong-import-order, wrong-import-position, ungrouped-imports
 
-try:
-  long        # Python 2
-except NameError:
-  long = int  # Python 3
```

Comment:
<!--new_thread; commit:cc2022157865d44e8dba070a766de78853c508bc; resolved:0-->
I think we need to keep `long` for Python 2 compatibility.

Reply:
The [int from the future.builtins](http://python-future.org/compatible_idioms.html#long-integers) is a subclass of python 2's long.